test/fixtures/linter/rules/renderer/8ControlRenderer.js

```diff
@@ -0,0 +1,20 @@
+sap.ui.define([], function () {
+	var myControlRenderer = {
+		// apiVersion: 1, // Must be present
```

Comment:
What does `Must be present` mean here? If it means `apiVersion: 1` must be present, I think that wouldn't be correct, as 2 must be present, right?